themes/default_symbols.fish

```diff
@@ -33,6 +33,9 @@ set FLSYM_ROOT_USER         "\u2192"
 # Symbol for VFISH segment
 set FLSYM_VFISH             "\u2635"
 
+# Symbol for CONDA segment
+set FLSYM_CONDA             "\U0001F40D"
```

Comment:
The `conda:` in the tty compatible is perfect, I might even change the one from `vfish` from # to `vfish:`. As `#` is used for by the `root` segment and we already have `git:` and `det:` for `git`.